tsl/src/continuous_aggs/create.c

```diff
@@ -368,6 +370,39 @@ cagg_add_trigger_hypertable(Oid relid, char *trigarg)
 	if (check_trigger_exists_hypertable(relid, CAGGINVAL_TRIGGER_NAME))
 		return;
 	ht = ts_hypertable_cache_get_cache_and_entry(relid, CACHE_FLAG_NONE, &hcache);
+	if (hypertable_is_distributed(ht))
+	{
+		DistCmdResult *result;
+		List *data_node_list = ts_hypertable_get_data_node_name_list(ht);
+		const char **cmds; /* same order as ht->data_nodes */
+		ListCell *cell;
+
+		int i = 0;
+		cmds = palloc(list_length(data_node_list) * sizeof(*cmds));
+		foreach (cell, ht->data_nodes)
+		{
+			HypertableDataNode *node = lfirst(cell);
+			StringInfo command = makeStringInfo();
+			appendStringInfo(command,
+							"CREATE TRIGGER %s AFTER INSERT OR UPDATE OR DELETE ON %s.%s FOR EACH ROW EXECUTE FUNCTION %s.%s(%d)",
+							CAGGINVAL_TRIGGER_NAME,
+							quote_identifier(NameStr(ht->fd.schema_name)),
+							quote_identifier(NameStr(ht->fd.table_name)),
+							quote_identifier(INTERNAL_SCHEMA_NAME),
+							quote_identifier(CAGG_INVALIDATION_TRIGGER),
+							node->fd.node_hypertable_id);
+			cmds[i++] = command->data;
+			elog(INFO, "[%s] hypertable_id=%d node_hypertable_id=%d", node->fd.node_name.data, node->fd.hypertable_id, node->fd.node_hypertable_id);
```

Comment:
Maybe use `DEBUG2` instead? Will both allow us to control if it should be printed and also allow us to find and remove debug messages going forward.
```suggestion
			elog(DEBUG2, "[%s] hypertable_id=%d node_hypertable_id=%d", node->fd.node_name.data, node->fd.hypertable_id, node->fd.node_hypertable_id);
```